contracts/SureYouDo.sol

```diff
@@ -391,7 +393,7 @@ contract SureYouDo is AllowedTokensManager, ReentrancyGuard {
      * @notice Cancels a challenge by creator.
      * @param challengeId The challenge id.
      */
-    function cancelChallenge(uint64 challengeId) external {
+    function cancelChallenge(uint64 challengeId) external nonReentrant {
```

Comment:
Could you please extract the changes made to this line from this PR? You already have them in PR #1.